src/sparkx/EventCharacteristics.py

```diff
@@ -321,6 +321,7 @@ def generate_eBQS_densities_Milne_from_OSCAR_IC(
         n_sigma_y: Union[float, int],
         n_sigma_z: Union[float, int],
         sigma_smear: Union[float, int],
+        kernel: str,
```

Comment:
kernel should have a default value. "gaussian", I'd propose

Reply:
That's done now